### ISSUE TITLE
[Internal] Engineers: Make a plan for dependency updates

### ISSUE THREAD

**Issue by knipec** [techmatters/terraso-mobile-client#1487]
This is from the May retrospective conversation. Just making this to keep track of it.

**Comment by knipec** [comment]
@DerekCaelin Good q -- I feel like this is roughly the same priority as other tech debt we're doing, and since we already pulled in 20% for August, I'm kinda inclined to just not assign it to a sprint, and plop it as a "Pri 1" tech debt item that we can pick up later? I don't feel entirely settled in how to prioritize our many different tech debt items, but feel like maybe it's worth trying to focus in and not do all the things?

**Comment by DerekCaelin** [comment]
Works for me! I'll move this out of the July sprint, and let the engineers decide whether it rises to the level of "should do in August"

**Comment by paulschreiber** [comment]
Here's the plan:
- We create a `dependencies` branch
- Depdendabot uses this branch for updates
- Updates are grouped (i.e. expo*, react*, types,*) to reduce the total number of PRs
- PRs go in to a merge queue and get merged automatically if they pass checks
- Once the merge queue is empty, `dependencies` is merged into `main` (and then re-created)
